### DISCUSSION TITLE
Обновление конфигурации через ssh или REST

### DISCUSSION THREAD

**Discussion by vfeshenko** [mikopbx/Core#551] [Q&A]
Есть необходимость некоторой интеграции MikoPBX с конфигуратором.
Как настроить MikoPBX разобрался, пишу необходимые настройки через ssh в sqlite. Проблема в том, что не нашел конечной точки для запуска процесса генерирования конфигов для Asterisk. Что нужно дернуть через ssh или, может, через REST, что-бы сгенерировались конфиги для Астериска, а сам Астериск выполнил core reload?

**Comment by jorikfon** [comment]
Система сама это делает, если данные меняются через REST API или через API интерфейса, если вы напрямую в базу пишете и сами следите за тем что данные все соответствуют всем правилами во всех таблицах, нет битых ссылок, то надо или ребут делать или дернуть что-то в интерфейсе, можно перезаписать маршрут или изменить приоритет, это легко подсмотреть в отладчике браузера, когда делается AJAX запрос к backend.
